packages/nextjs/components/impact-vector/ImpactVectorGraph.tsx

```diff
@@ -121,7 +121,13 @@ export default function ImpactVectorGraph({ data }: { data: DataSet[] }) {
             tickMargin={10}
             scale={isLogarithmic ? logScale : "linear"}
             domain={["auto", "auto"]}
+            tickFormatter={abbreviateNumber}
             allowDataOverflow
+            label={
+              <Text x={0} y={0} dx={15} dy={150} offset={0} angle={-90}>
```

Comment:
Can we make the text a little bigger? 14px looks good to me.

Reply:
👍 . I also moved it in a little bit closer to the axis. After abbreviating the higher numbers it felt like there was a bit too much space.
<img width="837" alt="Screenshot 2024-03-06 at 6 58 00 AM" src="https://github.com/BuidlGuidl/impact-calculator/assets/22231097/cf53fbe1-83ca-4e51-87c0-c8afc5b336f6">